frontend/src/components/auth/Logout.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect, } from 'react';
+import Button from '@material-ui/core/Button';
+import { useHistory, Redirect, } from "react-router-dom";
+import axios from "axios";
+
+export default function Logout(props) {
```

Comment:
maybe `LogoutDialog`?

Reply:
LogoutButton is probably the most descriptive.

Reply:
I changed to LogoutButton, its most accurate for what is rendered.